src/assets/i18n/en.json

```diff
@@ -1,6 +1,8 @@
 {
     "EMAILADDRESS": "E-mail address",
```

Comment:
Ich empfehle, bei den Übersetzungs-Files für jede Komponente eine eigene Untergruppe zu machen, so wie die primeng-Untergruppe. Sonst kommt man mit den Übersetzungen irgendwann in Teufels Küche, wenn die Anwendung größer wird

Reply:
Für "generische" Wörter halte ich es nicht für nicht sinnvoll.